src/Symfony/Bundle/FrameworkBundle/Command/SecretsSetCommand.php

```diff
@@ -49,13 +49,18 @@ protected function configure()
             ->setDescription(self::$defaultDescription)
             ->addArgument('name', InputArgument::REQUIRED, 'The name of the secret')
             ->addArgument('file', InputArgument::OPTIONAL, 'A file where to read the secret from or "-" for reading from STDIN')
+            ->addOption('force', 'f', InputOption::VALUE_NONE, 'Re-set an existing secret\'s value.')
```

Comment:
```suggestion
            ->addOption('force-override', 'f', InputOption::VALUE_NONE, 'Re-set an existing secret\'s value.')
```
?

Reply:
@OskarStark I can reword of course 👍 
But I do prefer to wait for some answers on the PR meaning itself before continuing working on this :)